js/cryptocom.js

```diff
@@ -0,0 +1,1210 @@
+        await this.loadMarkets ();
+        const request = {
+            'instrument_name': this.marketId (symbol),
+        };
+        if (limit) {
+            request['depth'] = limit;
+        }
+        const response = await this.publicGetPublicGetBook (this.extend (request, params));
+        // {
+        //     "code":0,
+        //     "method":"public/get-book",
+        //     "result":{
+        //       "bids":[[9668.44,0.006325,1.0],[9659.75,0.006776,1.0],[9653.14,0.011795,1.0],[9647.13,0.019434,1.0],[9634.62,0.013765,1.0],[9633.81,0.021395,1.0],[9628.46,0.037834,1.0],[9627.6,0.020909,1.0],[9621.51,0.026235,1.0],[9620.83,0.026701,1.0]],
+        //       "asks":[[9697.0,0.68251,1.0],[9697.6,1.722864,2.0],[9699.2,1.664177,2.0],[9700.8,1.824953,2.0],[9702.4,0.85778,1.0],[9704.0,0.935792,1.0],[9713.32,0.002926,1.0],[9716.42,0.78923,1.0],[9732.19,0.00645,1.0],[9737.88,0.020216,1.0]],
+        //       "t":1591704180270
+        //     }
+        // }
+        const orderBook = this.safeValue (response, 'result');
+        return this.parseOrderBook (orderBook, symbol);
+    }
+
+    async fetchBalance (params = {}) {
+        await this.loadMarkets ();
+        const response = await this.privatePostPrivateGetAccountSummary (params);
+        // {
+        //     "id": 11,
+        //     "method": "private/get-account-summary",
+        //     "code": 0,
+        //     "result": {
+        //         "accounts": [
+        //             {
+        //                 "balance": 99999999.905000000000000000,
+        //                 "available": 99999996.905000000000000000,
+        //                 "order": 3.000000000000000000,
+        //                 "stake": 0,
+        //                 "currency": "CRO"
+        //             }
+        //         ]
+        //     }
+        // }
+        const data = this.safeValue (response, 'result', {});
+        const coinList = this.safeValue (data, 'accounts', []);
+        const result = { 'info': response };
+        for (let i = 0; i < coinList.length; i++) {
+            const balance = coinList[i];
+            const currencyId = this.safeString (balance, 'currency');
+            const code = this.safeCurrencyCode (currencyId);
+            const account = this.account ();
+            account['total'] = this.safeString (balance, 'balance');
+            account['free'] = this.safeString (balance, 'available');
+            account['used'] = this.safeString (balance, 'order');
+            result[code] = account;
+        }
+        return this.safeBalance (result);
+    }
+
+    async fetchOrder (id, symbol = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchOrder requires a `symbol` argument');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'order_id': id,
```

Comment:
I think we're missing a market id in this request, otherwise I'm not entirely sure why we would require a `symbol`, right?